src/covid_model_seiir_pipeline/pipeline/forecasting/data.py

```diff
@@ -71,21 +71,16 @@ def load_location_ids(self) -> List[int]:
     def load_regression_coefficients(self, draw_id: int) -> pd.DataFrame:
         return self._get_regression_data_interface().load_regression_coefficients(draw_id=draw_id)
 
-    def load_transition_date(self, draw_id: int) -> pd.Series:
+    def load_transition_date(self, draw_id: int) -> pd.MultiIndex:
         dates_df = self._get_regression_data_interface().load_date_file(draw_id=draw_id)
         dates_df['end_date'] = pd.to_datetime(dates_df['end_date'])
-        transition_date = dates_df.set_index('location_id').sort_index()['end_date'].rename('date')
-        return transition_date
+        return dates_df['end_date'].rename('date')
 
     def load_beta_regression(self, draw_id: int) -> pd.DataFrame:
-        beta_regression = self._get_regression_data_interface().load_regression_betas(draw_id=draw_id)
-        beta_regression['date'] = pd.to_datetime(beta_regression['date'])
-        return beta_regression
+        return self._get_regression_data_interface().load_regression_betas(draw_id=draw_id)
```

Comment:
I don't think you can sanely make these changes unless you guarantee the data you're loading is loaded from a typed file format like parquet.

If CSV is still available you should be using a method in `pandas.api.types` to check if you need to convert via `pd.to_datetime`

Reply:
The CSV marshall is looking for a 'date' column and converting before we get to this layer.  Any other date-like columns have to be handled explicitly (as is done in `load_transition_date` above).  Is there something else you're worried about?  Or do you think that's unsafe?

Reply:
If this one was always redundant than removing it is dandy. I noticed a **lot** of other similar removals, I didn't pay any attention to the column being renamed.

I perceive the risk of you loading CSV data to be extremely low. Nonetheless you've introduced change that could lead to an unintended regression were that to happen. This clashes with my "use types and do other things very carefully" hat which is why I brought it up.